scripts/CheckSecurityConfigs.s.sol

```diff
@@ -37,24 +37,24 @@ contract CheckSecurityConfigs is Script {
     /**
      * @notice The entrypoint function.
      */
-    function run() external {
-        VmSafe.DirEntry[] memory addressesJsonEntries = vm.readDir("superchain/extra/addresses/mainnet");
+    function run(string memory jsonDir, string memory network) external {
```

Comment:
The network should match the provided RPC URL's chain ID, and we currently do not verify that. Since the RPC URL is a required command line arg, we could remove both of these input args and key off `block.chainid`, which will be the value returned by the RPC URL, to enforce that automatically. This also improves UX by reducing the number of args needed. For example:
```solidity
string memory jsonDir;
string memory network;
if (block.chainid == 1) {
  jsonDir = "superchain/extra/addresses/mainnet";
  network = "mainnet";
} else if (block.chainid == 11155111) {
  jsonDir = "superchain/extra/addresses/sepolia";
  network = "sepolia";
} else {
  string memory chainId = vm.toString(block.chainid);
  revert(string.concat("Unsupported chain ID: ", chainId));
}
```

If we do need to support arbitrary `jsonDir` values, we can have another run method to support that, but this way the common case of `superchain/extra/addresses/<network>` becomes safer and simpler

Reply:
Thanks! Here's a version that's slightly simpler to maintain, of course either is ok:

```solidity
string memory network;
if (block.chainid == 1) network = "mainnet";
else if (block.chainid == 11155111) network = "sepolia";
else revert(string.concat("Unsupported chain ID: ", vm.toString(block.chainid));

string memory jsonDir = string.concat("superchain/extra/addresses/", network);
```